test/preview/components/fetchers/test_link_content_fetcher.py

```diff
@@ -99,27 +98,25 @@ def test_from_dict(self):
 
     @pytest.mark.unit
     def test_run_text(self):
```

Comment:
We need to add a test case for when there are multiple input URLs. Otherwise the `else` branch with the multithreading is not triggered.